Tactics/CSE.lean

```diff
@@ -144,8 +148,43 @@ def getState : CSEM State := get
 /-- Get the mutable state. -/
 def setState : State → CSEM Unit := set
 
+def State.ofConfig (cfg : CSEConfig) : State := {
+  currentFuelSearch := cfg.fuelSearch,
+  currentFuelEliminate := cfg.fuelEliminate,
+}
+
 def CSEM.run (val : CSEM α) (config : CSEConfig) : TacticM α :=
-   val.run' {} |>.run config
+   val.run' (State.ofConfig config) |>.run config
+
+def CSEM.hasFuelSearch? : CSEM Bool := do
+  return (← getState).currentFuelSearch > 0
```

Comment:
```suggestion
def CSEM.hasFuelSearch : CSEM Bool := do
  return (← getState).currentFuelSearch > 0
```

For the same reason as `dryRun`: the `has` prefix already communicates the Booleannes